tags-transformer/composer.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "rollbar-examples/tag-transformer",
+    "description": "A simple package to demonstrate how to use the transformer in a Rollbar PHP app to move extra data to the tags object in the payload.",
+    "type": "project",
+    "authors": [
+        {
+            "name": "Artur Moczulski",
+            "email": "artur.moczulski@gmail.com"
+        },
```

Comment:
Was Artur involved in this code?

Reply:
Uh. Kinda. I copied the source code from here https://github.com/rollbar/rollbar-php-examples/tree/master/transformer